src/app.py

```diff
@@ -1,12 +1,30 @@
 from utils.hosts import LOCAL
+from flask_cors import CORS
+import os
+
+
+def configure_cors(app, origin):
```

Comment:
excellent, thanks for putting this in a separate function. it'd be great if you could go the extra mile and use our [config reader](https://github.com/smartsdk/ngsi-timeseries-api/blob/master/src/utils/cfgreader.py) instead of plain python `os.env.get`. an example of how to use it is [over here](https://github.com/smartsdk/ngsi-timeseries-api/blob/77ed420c0a7532bcc13d941c0402f457cc40407a/src/translators/timescale.py#L81)